packages/tenant-process/src/model/domain/errors.ts

```diff
@@ -319,7 +317,7 @@ export function descriptorNotFoundInEservice(
 
 export function notValidMailAddress(address: string): ApiError<ErrorCodes> {
   return new ApiError({
-    detail: `mail address ${address} not valid`,
+    detail: `Mail address ${address} not valid`,
```

Comment:
Not related to this PR (you can open a different PR if you prefer) but we need to be sure that logs do not contain any personal information.

```suggestion
    detail: `Mail address not valid`,
```